pkg/api/server/register_images.go

```diff
@@ -12,9 +12,9 @@ import (
 func (s *APIServer) registerImagesHandlers(r *mux.Router) error {
 	// swagger:operation POST /images/create images createImage
 	//
-	// Create an image from an image
-	//
 	// ---
+	// summary: Create an image from an image
+	// description: Create an image by either pulling it from a registry or importing it.
```

Comment:
The parameters do not look correct, see https://docs.docker.com/engine/api/v1.40/#operation/ImageCreate. The docker docs are also incomplete as there is an additional `message` parameter (which will soon reach the docs).